settings.py

```diff
@@ -21,10 +21,15 @@
 # Background color
 BGCOLOR = '#fff'
 
-# Output dir
+# Subdir
 subdir = lambda *d: os.path.join(os.path.dirname(__file__), *d)
+
+# Input dir
```

Comment:
I don't think these comments are useful, can you either remove them or make them something that's not the same as the constant names?